docs/encoding.md

```diff
@@ -0,0 +1,118 @@
+# Encoding Component
+
+This components provides functions like `encode()` and `decode()` so that you can deal with XML just like you deal with JSON!
+By using these functions there is no need for ever using the `SimpleXML` library again!
+
+## Example
+
+```php
+use function VeeWee\Xml\Dom\Configurator\pretty_print;
+use function VeeWee\Xml\Dom\Configurator\utf8;
+use function VeeWee\Xml\Dom\Configurator\validator;
+use function VeeWee\Xml\Dom\Validator\xsd_validator;
+use function VeeWee\Xml\Encoding\decode as xml_decode;
```

Comment:
I assume it will mostly be used as `xml_decode` could make the functions default name.

Reply:
Yes, changed it